specification/deviceprovisioningservices/resource-manager/readme.md

```diff
@@ -211,11 +221,32 @@ input-file:
 
 ```
 
-If there are files that should not be in the `all-api-versions` set, 
+If there are files that should not be in the `all-api-versions` set,
 uncomment the  `exclude-file` section below and add the file paths.
 
 ``` yaml $(tag) == 'all-api-versions'
 #exclude-file: 
 #  - $(this-folder)/Microsoft.Example/stable/2010-01-01/somefile.json
 ```
 
+## Suppression
+
+``` yaml
+directive:
+  - suppress: PageableOperation
```

Comment:
All these three rules are warnings and doesn't need to be suppressed. It would not fail the lintDiff check. Can you remove this suppression section?